src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/callback/AttachScannedDocumentsValidator.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.callback;
+
+import com.google.common.collect.Sets;
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.Documents;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.DuplicateDocsException;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import static java.util.stream.Collectors.toSet;
+import static org.slf4j.LoggerFactory.getLogger;
+
+public final class AttachScannedDocumentsValidator {
+
+    private static final Logger log = getLogger(AttachScannedDocumentsValidator.class);
+
+    private AttachScannedDocumentsValidator() {
+        // utility class constructor
```

Comment:
It should be a normal component with non-static method

Reply:
after brief discussion i don't mind at all if its component or not. will convert to component ;)